### PR TITLE
Thiruvenkatam A3,B1,B2 and B3

### PR DESCRIPTION
**Pull request by ThiruvenkatamR**
I have pushed the assignments given today to this branch. Can you please review and merge it?

### CODE REVIEW

File: thiru/layout2.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html class="no-js">
```
Comment (iammadhan) [inline thiru/layout2.html]: remove no-js class

File: thiru/layout2.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Layout</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="layout2.css">
+</head>
+
+<body>
+    <div>
```
Comment (iammadhan) [inline thiru/layout2.html]: change it to section

File: thiru/layout2.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <img class="image" src="./image.jpg" alt="Image" />
```
Comment (iammadhan) [inline thiru/layout2.html]: give meaningful alt text

File: thiru/dropdown.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    <meta charset="utf-8">
+    <meta httdiv-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Dropdown</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="dropdown.css">
+</head>
+
+<body>
+    <nav class="nav">
```
Comment (iammadhan) [inline thiru/dropdown.html]: why you have put class="nav"?

File: thiru/dropdown.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    padding: 10px 0;
+    box-sizing: border-box;
+    padding-left: 10px;
+}
+
+.menu.services:hover .items {
+    display: block;
+}
+
+.items li:hover {
```
Comment (iammadhan) [inline thiru/dropdown.css]: reuse the next rule with this rule

File: thiru/layout2.html (hunk 1 above)
Comment (ThiruvenkatamR) [reply thiru/layout2.html]: removed

File: thiru/design-inline.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+header {
+    margin: 0 auto;
+    border: 2px solid black;
+    height: 18%;
+    padding: 30px;
+    box-sizing: border-box;
+    text-align: center;
+    margin-bottom: 1%;
```
Comment (iammadhan) [inline thiru/design-inline.css]: use shorthand property for margin

File: thiru/design-inline.css [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    border: 2px solid black;
+    height: 18%;
+    padding: 40px;
+    box-sizing: border-box;
+    text-align: center;
+    bottom: 0;
+    position: absolute;
+    width: 100%;
+    margin-bottom: 1%;
+    margin-top: 1%;
```
Comment (iammadhan) [inline thiru/design-inline.css]: use margin shorthand property

Comment (ThiruvenkatamR) [reply thiru/design-inline.css]: done

File: thiru/design-inline.css (hunk 6 above)
Comment (ThiruvenkatamR) [reply thiru/design-inline.css]: done

**Pull request merged**